backend/src/dfusion/drafts.mo

```diff
@@ -0,0 +1,200 @@
+import Bool "mo:base/Bool";
+import Debug "mo:base/Debug";
+import Hash "mo:base/Hash";
+import Option "mo:base/Option";
+import Principal "mo:base/Principal";
+import Result "mo:base/Result";
+import Text "mo:base/Text";
+import Time "mo:base/Time";
+import TrieMap "mo:base/TrieMap";
+import Iter "mo:base/Iter";
+import TrieSet "mo:base/TrieSet";
+
+import Types "./types";
+
+shared(init_msg) actor class Drafts(
+  owner_: Principal
+) = this{
+  public type Draft = {
+    id: Nat;
+    title: Text;
+    content: Text;
+    time: Time.Time;
+  };
+  public type SetConfigRequest = {
+    countLimit: ?Nat;
+    titleLimit: ?Nat;
+    contentLimit: ?Nat;
+  };
+  public type Config = {
+    var countLimit: Nat;
+    var titleLimit: Nat;
+    var contentLimit: Nat;
+  };
+  // init
+	private let owner: Principal = owner_;
+  // map from user to drafts
+  private stable var userDraftsArray: [(Principal, TrieSet.Set<Draft>)] = [];
+  private var userDraftsMap = TrieMap.fromEntries<Principal, TrieSet.Set<Draft>>(userDraftsArray.vals(), Principal.equal, Principal.hash);
+  private stable var counter: Nat = 0;
+  private stable var authEntries : [(Principal, Bool)] = [(owner, true)];
+	private var auths = TrieMap.fromEntries<Principal, Bool>(authEntries.vals(), Principal.equal, Principal.hash);
+
```

Comment:
add: auths.put(owner, true);

Reply:
line 40 has ensured owner is true in auths